Core/EventHandlers.lua

```diff
@@ -1325,12 +1325,14 @@ function R:OnEvent(event, ...)
 			end
 		end
 
-		-- Handle opening Forgotten Chest (Venthyr only chest for Silessa's Battle Harness in Revendreth, Shadowlands)
-		if Rarity.isFishing and Rarity.isOpening and Rarity.lastNode and (Rarity.lastNode == L["Forgotten Chest"]) then
-			local names = {"Silessa's Battle Harness"}
+		-- Handle opening Forgotten Chest (Shadowlands, Revendreth chest for Stony's Infused Ruby pet and Silessa's Battle Harness mount)
+		if Rarity.isFishing and Rarity.isOpening and Rarity.lastNode and (Rarity.lastNode == L["Forgotten Chest"])
+			and GetBestMapForUnit("player") ~= CONSTANTS.UIMAPIDS.STORMSONG_VALLEY -- Chest with the same name in Stormsong Valley
```

Comment:
Can't we do ``== CONSTANTS.UIMAPIDS.REVENDRETH`` here?

Reply:
I haven't unlocked the 3rd tier of the travel network yet which is what is needed to start looting these chests. I don't know if `GetBestMapForUnit("player")` will return only the Revendreth map ID or if there are some others, like the maps in Sinfall etc.
That is why I opted to check for Stormsong.

Reply:
Good point. I also can't check since I'm Night Fae and haven't done any Shadowlands stuff, so I guess we can leave it at this.